src/libraries/Common/src/System/Net/Http/Http3/Frames/Http3ErrorCode.cs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Probably swap these copyrights to MIT as that's what is being used in corefx.

Reply:
I agree, but I'm not sure enough of the technicalities to know if it's legal for us to change it. Our strategy with HPACK was to keep the original license from ASP.NET. @terrajobst any opinions?

Reply:
New code should use the local license regardless.

Reply:
cc: @richlander

Reply:
Go ahead ... make the license change.